### ISSUE TITLE
New-BcContainer reports that docker is not running besides docker being installed and running.

### ISSUE THREAD

**Issue by Sven-Niehus** [microsoft/navcontainerhelper#2237]
**Describe the issue**
New-BcContainer reports that docker is not running besides docker being installed and running. 

The issue seems to be reported, because New-BcContainer checks for a hardcoded process called `docker` to determine if docker is running. [Source.](https://github.com/microsoft/navcontainerhelper/blob/fd2ae4841b0fd8b6ad2abd465d4df7ef9f2e79db/ContainerHandling/New-NavContainer.ps1#L475-L478)
A fix would be looking for a running `dockerd.exe` process or run something like `docker ps` to check if it will return an error.

I have already created a PR with the first fix: #2238 

**Scripts used to create container and cause the issue**
```
$containerName = 'bcserver'
$credential = Get-Credential -Message 'Using UserPassword authentication. Please enter credentials for the container.'
$auth = 'UserPassword'
$artifactUrl = Get-BcArtifactUrl -type 'OnPrem' -country 'de' -select 'Latest'
New-BcContainer `
    -accept_eula `
    -containerName $containerName `
    -credential $credential `
    -auth $auth `
    -artifactUrl $artifactUrl `
    -updateHosts
```

**Full output of scripts**
```
BcContainerHelper version 3.0.0
BcContainerHelper emits usage statistics telemetry to Microsoft
BcContainerHelper is version 3.0.0
BcContainerHelper is running as administrator
Host is Microsoft Windows Server 2022 Standard Evaluation - ltsc2022
New-BcContainer Telemetry Correlation Id: b11073d7-77bc-4e7f-b60f-52ba7b70a568
Docker Service not found. Docker is not started, not installed or not running Windows Containers.
At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\3.0.0\ContainerHandling\New-NavContainer.ps1:477 char:9
+         throw "Docker Service not found. Docker is not started, not i ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Docker Service ...ows Containers.:String) [], RuntimeException
    + FullyQualifiedErrorId : Docker Service not found. Docker is not started, not installed or not running Windows Containers.

```

**Screenshots**

**Additional context**
```
PS > docker images
REPOSITORY                          TAG              IMAGE ID       CREATED        SIZE
hello-world                         latest           fedae85b8e22   5 weeks ago    295MB
mcr.microsoft.com/businesscentral   10.0.20348.288   ad2f3c6fc8a8   2 months ago   7.81GB
```
```
PS > docker run hello-world
Hello from Docker!
This message shows that your installation appears to be working correctly.

To generate this message, Docker took the following steps:
 1. The Docker client contacted the Docker daemon.
 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
    (windows-amd64, nanoserver-ltsc2022)
 3. The Docker daemon created a new container from that image which runs the
    executable that produces the output you are currently reading.
 4. The Docker daemon streamed that output to the Docker client, which sent it
    to your terminal.

To try something more ambitious, you can run a Windows Server container with:
 PS C:\> docker run -it mcr.microsoft.com/windows/servercore:ltsc2019 powershell

Share images, automate workflows, and more with a free Docker ID:
 https://hub.docker.com/

For more examples and ideas, visit:
 https://docs.docker.com/get-started/
```

```
PS > Get-Process *docker*
Handles  NPM(K)    PM(K)      WS(K)     CPU(s)     Id  SI ProcessName                                                                                                    
-------  ------    -----      -----     ------     --  -- -----------                                                                                                    
    421      22    32564      47804      23.55   3884   0 dockerd                                                                                                        

PS > Get-Service *docker*
Status   Name               DisplayName                           
------   ----               -----------                           
Running  dockerd            dockerd
```

**Comment by freddydk** [comment]
Agree with your suggested fix, thanks

**Comment by freddydk** [comment]
Fixed in 3.0.1
